client/components/experiment/index.tsx

```diff
@@ -23,6 +24,11 @@ export { default as LoadingVariations } from './loading-variations';
  */
 export const Experiment: FunctionComponent< ExperimentProps > = ( props ) => {
 	const { isLoading: loading, variation, children } = props;
+
+	// we deliberately don't include any props/information about the experiment. This event is not intended to be used
+	// in an experiment as an exposure event.
+	recordTracksEvent( 'calypso_experiment_rendered', {} );
```

Comment:
There's no harm in including the experiment name. :slightly_smiling_face: Can you please add an explanation why we _have_ to fire this event (i.e., to generate the anonId)? Otherwise, we may move it later by mistake.